### PR TITLE
jormun: restrict bragi autocompletion to the coverage

### PR DESCRIPTION
**Pull request by kinnou02**
When doing a /places on a coverage if no shape as been provided by the
user we use the shape of the coverage to prevent showing objects that
are outside of the coverage.
The shape of the coverage is simplified to speedup the search.
An evolution has to be made on bragi to not apply the shape's filter on
stop areas.
Refer to https://jira.kisio.org/browse/COR-19

### CODE REVIEW

**Comment by kinnou02** [comment]
Jenkins, test this please.

**Review by crocme10 — APPROVED** [review]
Looks good

File: source/jormungandr/tests/authentication_tests.py [hunk 1]
```diff
@@ -551,7 +551,7 @@ def test_places_authentication(self):
         # but when querying /v1/coverage/<something>/places only one pt_dataset is given to bragi
         with user_set(app, FakeUserAuth, 'super_user'):
             with requests_mock.Mocker() as m:
-                m.get(requests_mock.ANY, json=response)
+                m.post(requests_mock.ANY, json=response)
```
Comment (AurelienLP) [inline source/jormungandr/tests/authentication_tests.py:554]: Why is it a post now ?

**Review by AurelienLP — APPROVED** [review]

File: source/jormungandr/tests/authentication_tests.py (hunk 1 above)
Comment (kinnou02) [reply source/jormungandr/tests/authentication_tests.py:554]: we use POST to send a shape in the body of the request

**Review by pbougue — APPROVED** [review]

**Pull request merged**